check/ts-build

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Build the bundle files for Cirq-web
+#
+# Usage:
+#     check/ts-build
+################################################################################
+
+# Get the working directory to the repo root.
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$(git rev-parse --show-toplevel)"
+
+cd 'cirq-web/cirq_ts'
+npx webpack --mode production
```

Comment:
we typically forward the arguments to scripts. Add `$@` here. 
You can test whether it works by `check/ts-build --watch` for example.